app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -61,16 +61,6 @@ def allowed_customization_templates(options = {})
     end
   end
 
-  def allowed_hosts_obj(options = {})
```

Comment:
@borod108 I do not thing that removing this method is going to give the desired results.  The base method [miq_request_workflow.rb#allowed_hosts_obj](https://github.com/ManageIQ/manageiq/blob/25cc66a96838f1cd04a5a41c2f75aaa8afc0796c/app/models/miq_request_workflow.rb#L1061) will return all hosts for all datacenters.  You only want to return the hosts that reside in the same datacenter as the template.